app/sidekiq/decision_review/submit_upload.rb

```diff
@@ -33,13 +38,6 @@ def perform(appeal_submission_upload_id)
       log_success(internal_id: appeal_submission.id, lighthouse_uuid: appeal_submission.submitted_appeal_uuid,
                   lighthouse_evidence_uuid: lh_upload_id, appeal_type: appeal_submission.type_of_appeal)
       StatsD.increment("#{STATSD_KEY_PREFIX}.success")
-    rescue => e
-      handle_error(e)
-    end
-
-    def handle_error(e)
-      StatsD.increment("#{STATSD_KEY_PREFIX}.error")
-      raise e
```

Comment:
I think it might make sense to keep this here, but just change the statsD key... Like it may be nice to still know/track the number of failed retry attempts (in addition to the exhaustion attempts that you added), but seperatly. Like changing line 41 from 
``` 
StatsD.increment("#{STATSD_KEY_PREFIX}.error")
```
To like 
```
StatsD.increment("#{STATSD_KEY_PREFIX}.attempt.error")
```
Or.. something. 

Up for discussion though!

Reply:
ooh yeah that could be useful — i talked to @data-doge about this briefly and he suggested possibly looking into whether we can pass metadata to the `success` StatsD and include the number of retries, but that it's not a blocker. maybe i could look into it as a follow up?

Reply:
Nah I dont think it is a blocker. And you could just log that data right before or after making the statsD call. The statsD call just increments a counter, at the key specified, it does not support additional data. But you can use the rails log to log whatever info you want, like the relevant details